libtiledbsoma/include/tiledbsoma/managed_query.h

```diff
@@ -146,6 +146,10 @@ class ManagedQuery {
         query_->set_condition(qc);
     }
 
+    void set_layout(tiledb_layout_t layout) {
```

Comment:
Please add a brief comment similar to others in this file. They're handy for code editors that read the comments.

Reply:
good catch @gspowley, updated